Nilesh

rshingavi97 opened the pull request:
Kindly review the changes and merge the code

Commit:
Implement solar system

Commit:
Solar system

Commit:
resolved merge conflit

Commit:
Removed unnecessery file SolarRender.h

Commit:
removed unnecessary files like Resource.rc

rshingavi97 committed:
Implement all review suggestions by Nilesh

mknilesh2016 commented on src/features/SolarSystem/SolarRender.cpp:
```diff
@@ -0,0 +1,297 @@
+#include <gl\GL.h>
+#include <gl\GLU.h>
+/* file for SOIL library to load the image*/
+#include "..\tools\soil\src\SOIL.h"
```
any reasons for not using direct include?  You should be able to use this - 

#include <SOIL.h>

mknilesh2016 commented on src/features/SolarSystem/SolarRender.cpp:
```diff
@@ -0,0 +1,297 @@
+	g_Uranus = 0.0f;
+	g_Neptune = 0.0f;
+	g_Angle = 0.0f;
+	LoadTexture(&gbTexture[0], "\\git\\project\\src\\features\\SolarSystem\\earth.bmp");
```
Looks to be hardcoded path so might not work for everybody. 

Let me tune the code a bit to be able to locate these resources.

mknilesh2016 reviewed (commented):
Thanks for the updated changes.  There are some formatting nits which i'll do on your behalf before i merge the changes to master.

Commit:
Implement review changes before merging pull request

- Corrected tabs to spaces to correct diff errors.
- Deleted textures on uninitialize()
- Added code to render SolarSystem() scene for 30 secs instead of
infinite
- Created res folder to contain resources used by the features
- Added script to copy resources to out folder after build to be picked
up by the application.
- changed all hardcoded paths to relative to be able to pick-up
- fixed a framework bug where main loop was not exitted on close button
click. This is done by setting m_exitLoop=true; on WM_DESTROY
